recipes/pshmem/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pshmem" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aba14aaf0862b4c56947762417a5500f2ad76d67c7789957258978539dfd7c68
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - numpy
```

Comment:
This shouldn't be needed in host for noarch python packages
```suggestion
```

Reply:
Ok, thanks.  Removed.